content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -103,147 +109,143 @@ The StatefulSet manifest, included below, creates a Cassandra ring that consists
-      kind: StatefulSet
-      metadata:
-        creationTimestamp: 2016-08-13T18:40:58Z
-        generation: 1
-        labels:
-          app: cassandra
-        name: cassandra
-        namespace: default
-        resourceVersion: "323"
-        selfLink: /apis/apps/v1/namespaces/default/statefulsets/cassandra
-        uid: 7a219483-6185-11e6-a910-42010a8a0fc0
-      spec:
-        replicas: 3
-      ```
-
-1. Change the number of replicas to 4, and then save the manifest. 
+    ```shell
+    kubectl edit statefulset cassandra
+    ```
+
+    This command opens an editor in your terminal. The line you need to change is the `replicas` field. The following sample is an excerpt of the StatefulSet file:
+
+    ```yaml
+    # Please edit the object below. Lines beginning with a '#' will be ignored,
+    # and an empty file will abort the edit. If an error occurs while saving this file will be
+    # reopened with the relevant failures.
+    #
+    apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
+    kind: StatefulSet
+    metadata:
+      creationTimestamp: 2016-08-13T18:40:58Z
+      generation: 1
+      labels:
+      app: cassandra
+      name: cassandra
+      namespace: default
+      resourceVersion: "323"
+      selfLink: /apis/apps/v1/namespaces/default/statefulsets/cassandra
+      uid: 7a219483-6185-11e6-a910-42010a8a0fc0
+    spec:
+      replicas: 3
+    ```
+
+2. Change the number of replicas to 4, and then save the manifest. 
 
    The StatefulSet now contains 4 Pods.
 
 1. Get the Cassandra StatefulSet to verify:
 
-      ```shell
-      kubectl get statefulset cassandra
-      ```
+    ```shell
+    kubectl get statefulset cassandra
+    ```
 
-      The response should be
+    The response should be
 
-      ```
-      NAME        DESIRED   CURRENT   AGE
-      cassandra   4         4         36m
-      ```
+    ```
+    NAME        DESIRED   CURRENT   AGE
+    cassandra   4         4         36m
+    ```
       
 {{% /capture %}}
 
 {{% capture cleanup %}}
 Deleting or scaling a StatefulSet down does not delete the volumes associated with the StatefulSet. This ensures safety first: your data is more valuable than an auto purge of all related StatefulSet resources. 
 
 {{< warning >}}
-**Warning:** Depending on the storage class and reclaim policy, deleting the Persistent Volume Claims may cause the associated volumes to also be deleted. Never assume you’ll be able to access data if its volume claims are deleted.
+**Warning:** Depending on the storage class and reclaim policy, deleting the PersistentVolumeClaims may cause the associated volumes to also be deleted. Never assume you’ll be able to access data if its volume claims are deleted.
 {{< /warning >}}
 
-1. Run the following commands to delete everything in a `StatefulSet`:
+1. Run the following commands (chained together into a single command) to delete everything in the Cassandra StatefulSet:
```

Comment:
Initially I didn't like the parenthetical, but the more I think about it, the more I like it. 👍 Good choice!